docs/asf.md

```diff
@@ -0,0 +1,888 @@
+* private@pouchdb.incubator.apache.org
+* commits@pouchdb.incubator.apache.org
+
+#### Subversion Directory:
+
+It is conventional to use all lower case, dash-separated (-) directory names.
+The directory should be within the incubator directory space
+(http://svn.apache.org/repos/asf/incubator).
+
+Example (OpenJPA):
+> https://svn.apache.org/repos/asf/incubator/openjpa
+
+PouchDB:
+n/a
+
+#### Git Repositories:
+It is conventional to use all lower case, dash-separated (-) repository names.
+The repository should be prefixed with incubator and later renamed assuming the
+project is promoted to a TLP.
+
+Example (Batchee):
+> https://gitbox.apache.org/repos/asf/incubator-batchee.git
+
+PouchDB:
+* https://gitbox.apache.org/repos/asf/incubator-pouchdb.git
+* https://github.com/apache/incubator-pouchdb.git
+
+#### Issue Tracking:
+
+Apache runs JIRA and Bugzilla. Choose one. Indicate the name by which project
+should be known in the issue tracking system.
+
+Example (OpenJPA):
+> JIRA Open-JPA (OPEN-JPA)
+
+PouchDB:
+GitHub Issues
+
+#### Other Resources:
+
+Describe here any other special infrastructure requirements necessary for the
+proposal. Note that the infrastructure team usually requires a compelling
+argument before new services are allowed on core hardware. Most proposals should
+not require this section.
+
+Most standard resources not covered above (such as continuous integration)
+should be added after bootstrapping. The infrastructure documentation explains
+the process.
+
+PouchDB:
+PouchDB makes significant use of GitHub Actions for CI. For 2024, we have used
+214,556 total minutes across 53,831 job runs. This can be migrated if need be
+but it’d be great if this carefully crafted setup could remain in place.
+
+### Initial Committers
+
+List of committers (stating name and an email address) used to bootstrap the
+community. Mark each which has submitted a contributor license agreement (CLA).
+Existing committers should use their apache.org email address (since they
+require only appropriate karma). Others should use the email address that is (or
+will be) on the CLA. That makes it easy to match CLAs with proposed committers
+to the project.
+
+It is a good idea to submit CLAs at the same time as the proposal. Nothing is
+lost by having a CLA on file at Apache but processing may take some time.
+
+Note this and this. Consider creating a separate section where interested
+developers can express an interest (and possibly leave a brief introduction) or
+ask them to post to the general list.
+
+Example (OpenJPA):
+> Abe White (awhite at bea dot com)
+> Marc Prud'hommeaux (mprudhom at bea dot com)
+> Patrick Linskey (plinskey at bea dot com)
+> ...
+> Geir Magnusson Jr (geirm at apache dot org) *
+> Craig Russell (clr at apache dot org) *
+
+PouchDB:
+
+Alba Herrerías Ramírez albaherreriasdev at gmail dot com
+Alex Anderson alexanderandersonofandover at gmail dot com
+Diana Thayer garbados at apache dot org (CLA)
```

Comment:
garbados at gmail dot com

Reply:
I prefer to go by Diana Belle or Diana Fernanda Belle, unless this is a matter of legal documentation.

Reply:
It doesn't go to my inbox so I can't read anything sent there. I can see about resolving that today.

Reply:
OK, resolved. It should forward to my email appropriately now.